corehq/apps/reports/forms.py

```diff
@@ -161,11 +162,19 @@ class ScheduledReportForm(forms.Form):
     )
 
     def __init__(self, *args, **kwargs):
+        super(ScheduledReportForm, self).__init__(*args, **kwargs)
+
         self.helper = FormHelper()
         self.helper.form_class = 'form-horizontal'
         self.helper.form_id = 'id-scheduledReportForm'
         self.helper.label_class = 'col-sm-3 col-md-2'
         self.helper.field_class = 'col-sm-9 col-md-8 col-lg-6'
+
+        domain = kwargs.get('initial', {}).get('domain', None)
+        if domain is not None and HOURLY_SCHEDULED_REPORT.enabled(domain, NAMESPACE_DOMAIN):
+            self.fields['interval'].choices.insert(0, ("hourly", "Hourly"))
```

Comment:
I think `"Hourly"` needs to be translated?

Reply:
@proteusvacuum Good catch!